docs/source/quick_start/introduction.rst

```diff
@@ -0,0 +1,42 @@
+
+What is MCDR
+============
+
+MCDReforged (MCDR, hereinafter) is a tool to manage your Minecraft server with the custom plugin system. No need to modify the original Minecraft server at all
+
+In-game calculator, player high-light, scoreboard manipulate, structure file management, backup / rollback... you can control the whole world with MCDR and its plugins
+
+Greatly thanks to `chino_desu <https://github.com/kafuuchino-desu>`__ and his `MCDaemon 1.0 <https://github.com/kafuuchino-desu/MCDaemon>`__ for the idea of such a cool tool
+
+Advantages
+----------
+
+Vanilla
+~~~~~~~
+
+MCDR runs above the server. It doesn't need to modify the server at all, which keep everything **vanilla**
+
+Plugin System
+~~~~~~~~~~~~~
+
+MCDR has a rich ecosystem of plugins from the community
+
+Meanwhile it's hot-reloadable that you don't need to shut down the server to update the plugins
+
+Compatibility
+~~~~~~~~~~~~~
+
+We supports most popular server softwares (Vanilla, Fabric, Spigot, Paper, etc.)
+
+On multiple platforms (Windows, Linux, Mac, etc.)
+
+How it works
+------------
+
+**TL;DR:** MCDR works like robot that stares at the server console, respond to server output quickly, and inputting commands for users
+
+MCDR uses `Popen <https://docs.python.org/3/library/subprocess.html#subprocess.Popen>`__ to start the server software as a sub-process, so it has the ability to control the standard input / out stream of the server
```

Comment:
The concept of "software" is beyond a "program" or "executable", so saying "start the server software as a sub-process" is confusing

```suggestion
MCDR uses `Popen <https://docs.python.org/3/library/subprocess.html#subprocess.Popen>`__ to start the server as a sub-process, so it has the ability to control the standard input / out stream of the server
```